runtime/src/main/java/org/corfudb/runtime/GarbageInformer.java

```diff
@@ -228,13 +235,13 @@ public void sendGarbageBatch(GarbageBatch batch) {
 
             // send GarbageEntry batch.
             stripIndexToGarbageEntries.forEach((stripeIndex, garbageEntries) ->
-                    rt.getAddressSpaceView().sparseTrim(rt.getLayoutView().getRuntimeLayout(), stripeIndex,
+                    rt.getAddressSpaceView().writeGarbage(rt.getLayoutView().getRuntimeLayout(), stripeIndex,
```

Comment:
Now we are sending garbage to all log units, do you still need stripIndex?

Reply:
It is still necessary. Garbage batch is sent to every log unit server in one stripe, not every log unit server.